Update docs to reflect new naming for UniFi integration (#20312)
home-assistant/home-assistant.github.io: source/_integrations/unifi.markdown

---
title: UniFi Network

> description: Instructions on how to configure UniFi integration with UniFi Controller by Ubiquiti.

Reviewer: ```suggestion
description: Instructions on how to configure UniFi integration with UniFi Network Controller by Ubiquiti.
```

---
title: UniFi Network

> description: Instructions on how to configure UniFi integration with UniFi Network Application by Ubiquiti.

Reviewer: ```suggestion
description: Instructions on how to configure UniFi integration with UniFi Network application by Ubiquiti.
```

The user needs administrator privileges in order to control switches.

> ### Understanding UniFi Naming

Reviewer: Put this in an FAQ at the bottom

> Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their Network Controller application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for UniFi Network application. It will not interface with devices for other applications like Protect, Talk, Access.**

Reviewer: ```suggestion
Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their Network Controller application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for UniFi Network application.**
```
Lets remove keywords that can be used when searching. to wrongfully find this integration

### Extra configuration of the integration

All configuration options are offered from the front end. Enter what UniFi Network integration you want to change options on and press the cog wheel. Some advanced options are available when "Advanced Mode" is enabled on your user profile page.

### Configuring Users

> The UniFi Network controller allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi Network device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.

Reviewer: ```suggestion
The UniFi Network application allows you to create multiple users on it besides the main administrator. If all you want to use is the device tracker then it is recommended that you create a limited user that has `read-only` permissions for the UniFi Network device tracker. If you want blocking of network access or POE control as well you would need to have 'admin' permissions.
```

## Presence detection

> This platform allows you to detect presence by looking at devices connected to a [Ubiquiti](https://ui.com/) [UniFi Network](https://unifi-network.ui.com/) controller. By default devices are marked as away 300 seconds after they were last seen.

Reviewer: ```suggestion
This platform allows you to detect presence by looking at devices connected to a [Ubiquiti](https://ui.com/) [UniFi Network](https://unifi-network.ui.com/) application. By default devices are marked as away 300 seconds after they were last seen.
```

### Troubleshooting and Time Synchronization

Presence detection depends on accurate time configuration between Home Assistant and the UniFi controller.

> If Home Assistant and the UniFi Network controller are running on separate machines or VMs ensure that all clocks are synchronized. Failing to have synchronized clocks will lead to Home Assistant failing to mark a device as home.

Reviewer: ```suggestion
If Home Assistant and the UniFi Network application are running on separate machines or VMs ensure that all clocks are synchronized. Failing to have synchronized clocks will lead to Home Assistant failing to mark a device as home.
```

### Service unifi.remove_clients

> Clean up clients on the UniFi Network Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.

Reviewer: ```suggestion
Clean up clients on the UniFi Network application that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```
```suggestion
Clean up clients on the UniFi Network Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```

> Clean up clients on the UniFi Network Controller that has only been associated with the controller for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.

Reviewer: ```suggestion
Clean up clients on the UniFi Network application that has only been associated with the Network application for a short period of time. The difference between first seen and last seen needs to be less than 15 minutes and the client can not have a fixed IP, hostname or name associated with it.
```

## Debugging integration

> If you have problems with UniFi Network or the integration you can add debug prints to the log.

Reviewer: ```suggestion
If you have problems with the UniFi Network application or integration you can add debug prints to the log.
```

    homeassistant.components.device_tracker.unifi: debug
    homeassistant.components.switch.unifi: debug
```

## FAQ

> ### Understanding UniFi Naming (UniFi Controller vs. UniFi Network Application)

Reviewer: ```suggestion
### Understanding UniFi Naming (UniFi Network application is the UniFi Controller)
```

*Nearly there* There are at least two instances of controller I couldn't add a comment on in MQTT and troubleshooting

Author: Yep, I only committed your suggestions. Going through it in depth now in my editor.

    homeassistant.components.device_tracker.unifi: debug
    homeassistant.components.switch.unifi: debug
```

## FAQ

### Understanding UniFi Naming (UniFi Controller vs. UniFi Network Application)

> Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their "UniFi Controller" application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. As a result, Ubiquiti has started rebranding "UniFi Controller" as the "UniFi Network Application". [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for the UniFi Network application.**

Reviewer: ```suggestion
Network management has always been Ubiquiti's main product and so UniFi for a while was always synonymous with their "UniFi Controller" application. However, UniFi has started branching and releasing other apps, like Protect, Talk and Access. As a result, Ubiquiti has started rebranding "UniFi Controller" as the "UniFi Network" application. [This post on the UniFi community](https://community.ui.com/questions/Clarifying-UniFi-Hardware-and-Software-Terminology/2557963a-e79d-4157-a78c-36d3f7b383fb) explains it pretty well. **This main UniFi Integration is _only_ for the UniFi Network application.**
```
